src/routes/GettingStartedPage/Wizard/InstallOptions.js

```diff
@@ -0,0 +1,146 @@
+
+const InstallOptions = ({
+  selectedEnv,
+  handleSelectedEnvChange,
+  selectedInstallation,
+  handleInstallationChange,
+}) => {
+  const [isSelectOpen, setIsSelectOpen] = React.useState(false);
+
+  const options = [
+    ...(selectedEnv === OPENSHIFT
+      ? [
+          <SelectOption key={0} value={OPERATOR}>
+            Operator (recommended)
+          </SelectOption>,
+        ]
+      : []),
+    <SelectOption key={1} value={HELM}>
+      Helm
+    </SelectOption>,
+  ];
+
+  const handleChange = (_, e) => {
+    handleSelectedEnvChange(e.target.value);
+    if (e.target.value != OPENSHIFT) {
+      handleInstallationChange(HELM);
+    } else {
+      handleInstallationChange(OPERATOR);
+    }
+  };
+
+  const onInstallationMethodSelect = (_, selection) => {
+    setIsSelectOpen(false);
+    handleInstallationChange(selection);
+  };
+
+  const onToggleSelect = () => {
+    setIsSelectOpen((prev) => !prev);
+  };
+
+  return (
+    <Stack hasGutter>
+      <StackItem>
+        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+          <FlexItem>
+            <Title headingLevel="h1">Installation Options</Title>
+          </FlexItem>
+          <FlexItem>
+            <HeaderExternalLink />
+          </FlexItem>
+        </Flex>
+      </StackItem>
+      <StackItem>
+        <TextContent>
+          <Text component={TextVariants.h3}>Select your environment</Text>
+        </TextContent>
+        <Radio
+          label="OpenShift"
+          id={OPENSHIFT}
+          name={OPENSHIFT}
+          value={OPENSHIFT}
+          aria-label={OPENSHIFT}
+          onChange={handleChange}
+          isChecked={selectedEnv === OPENSHIFT}
+        />
+        <Radio
+          label="EKS"
+          id={EKS}
+          name={EKS}
+          value={EKS}
+          aria-label={EKS}
+          onChange={handleChange}
+          isChecked={selectedEnv === EKS}
+        />
+        <Radio
+          label="AKS"
+          id={AKS}
+          name={AKS}
+          value={AKS}
+          aria-label={AKS}
+          onChange={handleChange}
+          isChecked={selectedEnv === AKS}
+        />
+        <Radio
+          label="GKE"
+          id={GKE}
+          name={GKE}
+          value={GKE}
+          aria-label={GKE}
+          onChange={handleChange}
+          isChecked={selectedEnv === GKE}
+        />
+      </StackItem>
+      <StackItem>
+        {selectedEnv && (
+          <>
+            <TextContent>
+              <Text component={TextVariants.h3}>
+                Select your installation method
+              </Text>
+            </TextContent>
+            <Select
+              variant="single"
+              isOpen={isSelectOpen}
+              onToggle={onToggleSelect}
+              selections={selectedInstallation}
+              onSelect={onInstallationMethodSelect}
+              aria-label="Select your installation method"
+              menuAppendTo={document.getElementById(WIZARD_ID)}
```

Comment:
There was an issue with the select dropdown living inside the policy wizard body which would cause an overflow when opened – requiring you to scroll down to select an option. This change attaches to the select dropdown to the top level wizard so that it can just overlap all other elements in the wizard.